src/analysis/processing/qgsalgorithmdensifygeometriesbycount.cpp

```diff
@@ -0,0 +1,124 @@
+/***************************************************************************
+                         qgsalgorithmdensifygeometries.cpp
+                         ---------------------
+    begin                : October 2019
+    copyright            : (C) 2019 by Clemens Raffler
+    email                : clemens dot raffler at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+
+#include "qgsalgorithmdensifygeometriesbycount.h"
+
+///@cond PRIVATE
+
+QString QgsDensifyGeometriesByCountAlgorithm::name() const
+{
+  return QStringLiteral( "densifygeometries" );
+}
+
+QString QgsDensifyGeometriesByCountAlgorithm::displayName() const
+{
+  return QObject::tr( "Densify by count" );
+}
+
+QStringList QgsDensifyGeometriesByCountAlgorithm::tags() const
+{
+  return QObject::tr( "add,vertex,vertices,points,nodes" ).split( ',' );
+}
+
+QString QgsDensifyGeometriesByCountAlgorithm::group() const
+{
+  return QObject::tr( "Vector geometry" );
+}
+
+QString QgsDensifyGeometriesByCountAlgorithm::groupId() const
+{
+  return QStringLiteral( "vectorgeometry" );
+}
+
+QString QgsDensifyGeometriesByCountAlgorithm::shortHelpString() const
+{
+  return QObject::tr( "This algorithm takes a polygon or line layer"
```

Comment:
Be careful of missing trailing spaces here -- this string is missing spaces between "layerand", "geometrieshave", etc.

Also would be good to add `\n\n` in places where the original help had a paragraph break